### PR TITLE
{toolchain} foss/2015b (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
depends on ~~#1836~~ (GNU), ~~#1694~~ + ~~#1841~~ (gompi/2015b)

**note**: do *not* consider the definition of this common toolchain final until the PR is merged!

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.14-GNU-5.1.0-2.25-LAPACK-3.5.0.eb [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+name = 'OpenBLAS'
+version = '0.2.14'
+
+lapackver = '3.5.0'
+versionsuffix = '-LAPACK-%s' % lapackver
+
+homepage = 'http://xianyi.github.com/OpenBLAS/'
+description = """OpenBLAS is an optimized BLAS library based on GotoBLAS2 1.13 BSD version."""
+
+toolchain = {'name': 'GNU', 'version': '5.1.0-2.25'}
```
Comment (JensTimmerman) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.14-GNU-5.1.0-2.25-LAPACK-3.5.0.eb]: where is the easyconfig for this toolchain?

Comment (JensTimmerman) [inline easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.14-GNU-5.1.0-2.25-LAPACK-3.5.0.eb]: nvmd, it's in https://github.com/hpcugent/easybuild-easyconfigs/pull/1669

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/237f32e35570c9a476c7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SLES 11_SP2, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/6e9c0ff95655a9b25b79 for a full test report.

**Comment by boegel** [comment]
updated to use OpenMPI 1.8.6 (just released)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/92b8aab5fbbf732d79ee for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux debian 7.8, Intel(R) Xeon(R) CPU            5130  @ 2.00GHz, Python 2.7.3
See https://gist.github.com/18a5bbe7de58ba297857 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SLES 11_SP2, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.6.9
See https://gist.github.com/13cf978ac9156a711e19 for a full test report.

**Comment by boegel** [comment]
@gjbex, @backelj, @hajgato: please test? (note: needs #1694)

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.5, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/75eb6be95cd0886bbf84 for a full test report.

**Comment by backelj** [comment]
Test report by @backelj
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/fad64dd2c1f864ba71f8 for a full test report.

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.4, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/6c84070e98dcdbf1f1ba for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/f634bd03cd861dddb2d9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b18ea719e00e053d793a for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.5, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/97c269f8c626002eb3b7 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a0b5622432bebbf75826 for a full test report.

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/131a4d3c2f9c92df31ab for a full test report.

**Comment by hajgato** [comment]
Test report by @hajgato
**SUCCESS**
Build succeeded for 20 out of 20 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.6.6
See https://gist.github.com/599e9cfbddaa37dca7e9 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/3c36bda8407efb88268e for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/84c933e4b7f8ef9c4656 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
Linux red hat enterprise linux computenode 6.5, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.6.6
See https://gist.github.com/d878c29d3ee2122ae803 for a full test report.

**Comment by boegel** [comment]
Good to go after extensive testing, thanks to all who helped review/test this one!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 14 out of 14 (5 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/867049fcd150e4c063f4 for a full test report.

**Pull request merged**